hadoop-hdds/container-service/src/test/java/org/apache/hadoop/ozone/container/checksum/TestContainerChecksumTreeManager.java

```diff
@@ -97,7 +98,7 @@ public void init() {
     container = mock(KeyValueContainerData.class);
     when(container.getContainerID()).thenReturn(CONTAINER_ID);
     when(container.getMetadataPath()).thenReturn(testDir.getAbsolutePath());
-    checksumFile = new File(testDir, CONTAINER_ID + ".tree");
+    checksumFile = new File(testDir, CONTAINER_ID + CONTAINER_DATA_CHECKSUM_EXTENSION);
```

Comment:
I deliberately hardcoded the path because changing the name of the file is somewhat of an incompatible change and I wanted `testChecksumTreeFilePath` to fail because of that. Maybe we can add a comment here explaining that in case such a change happens in the future.